### DISCUSSION TITLE
Webhook: create item event payload is missing field values that have a default value set

### DISCUSSION THREAD

**Discussion by goggi** [directus/directus#13591] [Feature Requests]
### Describe the Bug

Create item payload is missing if the field value has a default value set

![image](https://user-images.githubusercontent.com/1946339/170655703-368114bc-f506-4fb2-9e5d-d21690b1b301.png)

### To Reproduce

1. Create a new filed (String) that has a default value
 ![image](https://user-images.githubusercontent.com/1946339/170655409-61372aa9-1435-4913-ba6b-64858dd05efa.png)

2. Create a webhook that is triggered when items are created within the collection where you created the new field.
3. Create a new item in the collection where you created the filed,  dont make any changes to the new field
![image](https://user-images.githubusercontent.com/1946339/170655621-58475eab-3f83-4c72-8e83-8cf6b191641c.png)

### What version of Directus are you using?

9.11.0

### What version of Node.js are you using?

16.15.0

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
This is technically working as expected, as the webhook passes on the payload that was submitted by the user, not the full copy of the item that was just created. Treating this as a feature request to be able to send a copy of the full item in the webhook, rather than just the submitted payload 🙂  (Which is already included and going to be possible in #12522 😄 )

**Reply by goggi** [reply]
Thanks for the update!! I'm really looking forward to the Data Flow feature!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
